invenio_app_ils/ui/backoffice/src/App.js

```diff
@@ -2,21 +2,21 @@ import React, { Component } from 'react';
 
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
-import LoanDetails from './pages/LoanDetails';
-import ItemDetails from './pages/ItemDetails';
-import NotFound from './components/NotFound';
+import { LoanDetails } from './pages/LoanDetails';
+import { ItemDetails } from './pages/ItemDetails';
+import NotFound from './common/components/NotFound';
 
 import './App.scss';
 
 export default class App extends Component {
   render() {
     return (
-      <Router basename="/backoffice">
+      <Router>
```

Comment:
Didn't like the basename?

Reply:
We do, but there were some issues, where the backoffice was resolving to /, in the upcoming commits I am working with routes, so there is a possibility to be restored.